README.md

```diff
@@ -97,6 +97,10 @@ A number of other Homebrew formulas written by [Nixops](https://github.com/nixop
 
 A `.deb` package is available [here](https://github.com/burst-apps-team/burstcoin-packages/releases/tag/v2.3.0).
```

Comment:
not the last version `.deb` package

Reply:
Yep, I know. I will update this when I get a chance